pages/result.js

```diff
@@ -38,29 +41,53 @@ export default function Index() {
         router.replace('/sort');
       }
       const sorted = battles[battles.length - 1].result;
+      let oshi = sorted.slice(0,1)
+      setOshiName(oshi[0].name)
       setStatus(statusOptions[memberStatus - 1].label);
       setSortedTopThree(sorted.slice(0, 3));
       setSortedMembers(sorted.slice(3, sorted.length));
     }
+    
   }, [battles, currentMatchId, router, isMobileScreen, memberStatus]);
   if (!battles) {
     return <Loader />;
   }
+
+  function capture()
+  {
+    const containerToCapture = containerRef.current;
+    containerToCapture.style.backgroundColor = '#FDECEF'
+    
+    html2canvas(containerToCapture).then(function (canvas) {
+
+      const imageData = canvas.toDataURL('image/png'); 
+      
+      containerToCapture.style.backgroundColor = ''
+
+       const downloadLink = document.createElement('a');
+       downloadLink.href = imageData;
+       downloadLink.download = 'results.png'; 
+ 
+       downloadLink.click();
+    });
+  }
```

Comment:
Great Work!
I really appreciate your effort in this export result feature, this will really help the user to share their result.
unfortunately, there are some issues with this feature:
- the result rows seem to be a bit squashed together
- I can only download the result once from my phone, after that I can't download another result.

you can try out the preview of your feature right here
https://jkt48-member-sorter-git-fork-rksrbbn-main-adityaiwd.vercel.app/